src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -570,6 +570,7 @@ export class ExerciseAssessmentDashboardComponent implements OnInit {
                     if (this.unassessedSubmissionByRound) {
                         this.unassessedSubmissionByRound.delete(correctionRound);
                     }
+                    return;
```

Comment:
Why is this needed?

Reply:
Was actually a refactoring error by myself since before it was handled as an `error:`, meaning that the other code in `next:` didn't run